src/query_builder.erl

```diff
@@ -38,7 +39,7 @@ build_lookup_query(Collection, Metric, Grouping) ->
     {ok, Query, Values}.
 
 build_lookup_query(Bucket, Metric, Where, Grouping)
-  when is_list(Metric) ->
+  when is_list(Metric); is_atom(Metric) ->
```

Comment:
I think ; `Metric =:= undefined` is clearer, also guards against people passing un `undfined` like I'll frequently do!